src/coro/switch.c

```diff
@@ -35,11 +57,20 @@ void coro_routine_entry();
 
 __asm__(
     ".text\n"
+#if defined(__aarch64__)
+    ".globl coro_routine_entry\n"
+#endif
     "coro_routine_entry:\n"
 #if defined(__x86_64__)
     "pop %rdi\n"
     "pop %rcx\n"
     "call *%rcx\n"
+#elif defined(__aarch64__)
+    "mov x8, sp\n"
+    "ldp x0, x9, [x8], #16\n"
+    "mov sp, x8\n"
+    "blr x9\n"
+    "nop\n"
```

Comment:
Why does `nop` appear here?

Reply:
I forgot to remove this, I will remove it.

Reply:
removed nop